.github/workflows/tests.yml

```diff
@@ -188,6 +188,16 @@ jobs:
           docker tag brew "homebrew/ubuntu16.04:master"
           docker push "homebrew/ubuntu16.04:master"
 
+      - name: Deploy the retired Docker image to GitHub Packages and Docker Hub
+        if: github.ref == 'refs/heads/master'
+        run: |
+          printf 'FROM alpine\nCMD echo "Error: homebrew/ubuntu16.04:master is retired. Use homebrew/brew or homebrew/ubuntu16.04 or homebrew/ubuntu22.04:master"; exit 1\n' |
+            docker build --tag=retired -
+          docker tag retired "ghcr.io/homebrew/ubuntu16.04:master"
+          docker push "ghcr.io/homebrew/ubuntu16.04:master"
+          docker tag retired "homebrew/ubuntu16.04:master"
+          docker push "homebrew/ubuntu16.04:master"
```

Comment:
Seems weird to do this every time when it's something that only needs to be done once.

Reply:
We can revert this PR immediately after it's merged.

Reply:
How about guarding whatever we go with to the 3.6.0 tag? Would like to have the code kept around so we have it for the next Ubuntu LTS upgrade. Alternatively, could just add it to the docs.